python/cudf/cudf/core/dataframe.py

```diff
@@ -6171,6 +6176,46 @@ def _from_columns_like_self(
         result._set_column_names_like(self)
         return result
 
+    @_cudf_nvtx_annotate
+    def interleave_columns(self):
+        """
+        Interleave Series columns of a table into a single column.
+
+        Converts the column major table `cols` into a row major column.
+
+        Parameters
+        ----------
+        cols : input Table containing columns to interleave.
+
+        Examples
+        --------
+        >>> df = DataFrame([['A1', 'A2', 'A3'], ['B1', 'B2', 'B3']])
```

Comment:
This is required for doctests to pass.
```suggestion
        --------
        >>> import cudf
        >>> df = cudf.DataFrame([['A1', 'A2', 'A3'], ['B1', 'B2', 'B3']])
```